testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/ejb/remote/byteman/LastNodeToLeaveRemoteEJBTestCase.java

```diff
@@ -130,7 +130,7 @@ public LastNodeToLeaveRemoteEJBTestCase() throws Exception {
         public String selectNode(String clusterName, String[] connectedNodes, String[] totalAvailableNodes) {
             Set<String> connectedNodesSet = Arrays.asList(connectedNodes).stream().collect(Collectors.toSet());
             Set<String> totalNodesSet = Arrays.asList(totalAvailableNodes).stream().collect(Collectors.toSet());
-            LOGGER.infof("Calling ClusterNodeSelector.selectNode(%s,%s,%s)", clusterName, connectedNodesSet, totalNodesSet);
+            LOGGER.debugf("Calling ClusterNodeSelector.selectNode(%s,%s,%s)", clusterName, connectedNodesSet, totalNodesSet);
```

Comment:
@rachmatowicz Could you look at these changes? I am not sure how often are these needed / useful for debugging.

Reply:
this can be debug